client/src/components/Graph/D3Plot.vue

```diff
@@ -0,0 +1,307 @@
+<template>
+  <div id="d3-chart" class="plot"></div>
+</template>
+<script>
+import * as d3 from 'd3';
```

Comment:
Anyway we could have a different alias? Would be great if we didn't see 'd3' anywhere in the code beyond the imports -- could let the language match the use case whereas the name of the library tells us nothing about what it does.

Reply:
What would be the solution then? `d3` is short and does explain what library is being used when the term is entered. It is used a lot too.

Reply:
Also many libraries don't explain what it does. For example: `axios`.

Reply:
Fair enough on the 'axios' point -- I guess my main gripe is less with using library functions as d3.* and more with including the name of the library in the wrappers you have for it. Like 'D3Plot' as a component name, 'd3-plot' as an id name and d3plotData as an item in our store.

It's like if we created a thin wrapper for axios requests to make calling them a little easier and then proceeded to call the file axios-requests.js, the class in it AxiosRequests, and any time we had data we wanted to use as params called it axiosRequestData. It just ties the functionality / naming to the implementation instead of the content & context.

Does that make more sense?

Reply:
Ah that makes sense. Definitely. Will change the file name.

Reply:
And the other instances that it is referenced in that way.

Reply:
Sorry. One more comment. it was done that way because while developing this, it was done side-by-side with the chart.js implementation. So that is why it stuck around. Just didn't think about changing it until you mentioned it here. Thank you.